src/bicome/controllers/GameController.java

```diff
@@ -7,22 +7,20 @@
 import javafx.scene.image.ImageView;
 import javafx.scene.layout.AnchorPane;
 import javafx.scene.control.Label;
-import java.util.ResourceBundle;
 import javafx.scene.image.Image;
 import javafx.collections.FXCollections;
 import javafx.event.ActionEvent;
 import javafx.scene.input.MouseEvent;
-import javafx.scene.image.Image;
-import javafx.scene.image.ImageView;
+import javafx.scene.*;
+import javafx.scene.layout.StackPane;
+import javafx.scene.shape.Rectangle;
 
-import com.jfoenix.controls.*;
-import javafx.animation.RotateTransition;
-import javafx.event.ActionEvent;
 
-import javax.swing.*;
+public class GameController implements Initializable{
```

Comment:
Same with here. You need to implement `initialize(URL location, ResourceBundle resources)`

Reply:
I think we should only keep world from logic part. We don't need organism arrays. We can do this after cs lectures.